word_language_model/main.py

```diff
@@ -92,7 +109,17 @@ def batchify(data, bsz):
 ###############################################################################
 
 ntokens = len(corpus.dictionary)
-model = model.RNNModel(args.model, ntokens, args.emsize, args.nhid, args.nlayers, args.dropout, args.tied).to(device)
+if args.model == 'Transformer':
```

Comment:
Can we do this within Model? That is, you extend RNNModel to contain both and pass a flag to decide which one to use. This is in line with RNN settings GRU, LSTM, etc.

Reply:
I was thinking about merging TransformerSeq2Seq and RNNModel, but realized they are very different in the architecture.

Reply:
Let's talk about this in person.